src/Iterable/windowed.php

```diff
@@ -23,7 +23,7 @@
  * @param int $step The number of elements to move the window forward by on each step, by default 1
  */
 function windowed(
-    iterable $iterable,
+    iterable $collection,
```

Comment:
In the end, it is still an iterable right?

Reply:
Linking for reference https://github.com/JetBrains/kotlin/blob/30788566012c571aa1d3590912468d1ebe59983d/libraries/stdlib/common/src/generated/_Collections.kt#L3323

Reply:
Feels weird to have an iterable that cannot be converted to an array when arrays are also iterable.

It's terable!